terraform/browser/browser.tf.json.template.py

```diff
@@ -48,6 +48,50 @@
 provision_custom_search = False
 
 
+def content_security_policy() -> str:
+    return ';'.join([
+        "default-src 'self'",
```

Comment:
You are using two different mechanism to represent spaces. Inline and as a joiner. That's confusing and inconsistent. Use a Python function called `q` to quote values like `'self'` so you don't need to alternate between quote types, and can use single quotes uniformly.